book/src/continuous_integration/travis.md

```diff
@@ -0,0 +1,25 @@
+# Travis CI
+
+You can add Clippy to Travis CI in the same way you use it locally:
+
+```yml
+language: rust
+rust:
+  - stable
+  - beta
+before_script:
+  - rustup component add clippy
+script:
+  - cargo clippy
+  # if you want the build job to fail when encountering warnings, use
+  - cargo clippy -- -D warnings
+  # in order to also check tests and non-default crate features, use
+  - cargo clippy --all-targets --all-features -- -D warnings
+  - cargo test
+  # etc.
+```
+
+Note that adding `-D warnings` will cause your build to fail if **any** warnings are found in your code.
```

Comment:
Probably worth copying this to all configurations? Or for DRY purposes, this should probably be moved to the usage part?

Reply:
Agree with moving it to usage, and then maybe in the top level of `continuous_integration.md` we say something like "check the usage docs for all of the command line options".